pkg/blobstore/existence_precondition_blob_access.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/buildbarn/bb-storage/pkg/blobstore"
 	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
 	"github.com/buildbarn/bb-storage/pkg/util"
+	"go.opencensus.io/trace"
```

Comment:
Nit: current code style is to place an empty line when imports come from different domain names. Domains are also sorted alphabetically. So in this case, opencensus.io should go below golang.org.